portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +25,46 @@ body {
   max-width: 100%;
 }
 
+#navigation {
+  font-size: 5vw;
+  display:grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows:1fr 1fr;
+  color:white;
+}
+
+.navigation-item {
+    display:flex;
+    flex-direction:column;
+    align-items:center;
```

Comment:
[Alphabetize declarations.](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

Reply:
Attributes alphabetised in commit ab8c4a6